host/src/connection.rs

```diff
@@ -282,4 +258,13 @@ impl<'stack> Connection<'stack> {
             Err(e) => Err(e),
         }
     }
+
+    /// Transform BLE connection into a `GattConnection`
+    #[cfg(feature = "gatt")]
+    pub fn to_gatt<'values, 'server, M: RawMutex, const AT: usize, const CT: usize, const CN: usize>(
```

Comment:
I'd like to rename this `with_attribute_server` to signify that you want to handle this connection with an attribute server.